src/sonic-config-engine/sonic-cfggen

```diff
@@ -266,9 +310,7 @@ def main():
         if  brkout_table is not None:
             deep_update(data, {'BREAKOUT_CFG': brkout_table})
 
-    for json_file in args.json:
-        with open(json_file, 'r') as stream:
-            deep_update(data, FormatConverter.to_deserialized(json.load(stream)))
+    __process_json(args, data)
```

Comment:
>__process_json [](start = 4, length = 14)

If you use leading underscore to mark function as private, suggest use single underscore.

Reply:
ACK. I also plan to do some refactoring in a subsequent PR